core/modules/order/components/UserOrdersHistory.ts

```diff
@@ -26,6 +27,13 @@ export default {
   methods: {
     onBottomScroll () {
       ++this.pagination.current
-    }
+      let total_count = this.$store.state.user.orders_history.total_count ? this.$store.state.user.orders_history.total_count : 0
```

Comment:
In PWA, we prefer camelCase - please use it. In the object we have snake_case only because it comes directly from M2. Also please change it to const as it never changes

Reply:
modified as per your suggestion